pom.xml

```diff
@@ -821,7 +821,8 @@
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-deploy-plugin</artifactId>
               <configuration>
-                <updateReleaseInfo>true</updateReleaseInfo>
```

Comment:
This was removed because this is no longer a valid configuration option for this plugin (since ???).